interactions/api/http/client.pyi

```diff
@@ -2,15 +2,41 @@ from typing import Optional, Tuple
 
 from .request import _Request
 from ...api.cache import Cache
+from .channel import _ChannelRequest
+from .emoji import _EmojiRequest
+from .guild import _GuildRequest
+from .interaction import _InteractionRequest
+from .member import _MemberRequest
+from .message import _MessageRequest
+from .reaction import _ReactionRequest
+from .request import _Request
+from .route import Route
```

Comment:
The `Route` import isn't necessary here.

Reply:
Done